tests/dns/dns-rrtype/test.rules

```diff
@@ -0,0 +1,8 @@
+# Only alert on requests.
+alert dns any any -> any any (dns.rrtype:1; flow:to_server; sid:1; rev:1;)
+
+# Only alert on responses.
+alert dns any any -> any any (dns.rrtype:1; flow:to_client; sid:2; rev:1;)
+
+# Negated rrtype value
+alert dns any any -> any any (dns.rrtype:!1; flow:to_client; sid:3; rev:1;)
```

Comment:
what all rrtypes are in your pcap?

Reply:
they're all `rrtype: 1`

Reply:
and, what is your expectation from this rule then?

Reply:
I found another input.pcap that has two dns packets with rrtype: 1 and rrtype: 5 in the answer message. Should I try that out instead?

Reply:
Not required, no. Try answering the following and you might figure it out 😉 
1. what is your expectation from each rule? e.g. rule 1 should alert because there is a rrtype 1 in the pcap on a pkt in to_server dir, etc
2. if there is just one rrtype in the pcap, and it is 1, can it be 1 and not 1?

Reply:
got it, I've used a different pcap file and the negated value test is passing. Creating a new PR now.

Reply:
Sure but that's not what I meant :)
If the pcap has just rrtype 1, then, the rule that alerts on check for `rrtype: 1` should alert rightly so.
But, it's incorrect to also add and expect a rule that says `rrtype: !1` to alert on the same pcap. It should never alert because there is no other rrtype than 1 in the pcap.
So, what you could have done is made the rule with `sid:3` check for any other rrtype in negation. For the given pcap, `rrtype: !2`, `rrtype: !10`, `rrtype: !5` or whatsoever should all match as neither of these numbers are 1 which is what's present in the pcap. 😉

Reply:
ah that makes so much sense. I'll try this out and see if the negated tests pass. Thanks for guiding me!